source/javascripts/_componentRegister.js

```diff
@@ -12,7 +12,7 @@ angular
 		"rStepItem",
 		register(StepItem, [
 			"step",
-			"title",
+			"displayName",
```

Comment:
sorry don't have context here, can you elaborate why `displayName` is better than `title`? what are the differences?

Reply:
Since `title` is a reserved/existing HTML attribute, which is used to add tooltip info to an element, it showed up accidentally whenever hovering over the step with the wrong message. (step.displayName() since that was the input prop fro the react element)
Basically, it is the fix for https://bitrise.atlassian.net/browse/WEB-3943

Reply:
ahh nice, good catch ok 👍